### PR TITLE
ZEPPELIN-206: provide shell genthrift.sh to generate java code and auto insert license header

### PR DESCRIPTION
**Pull request by caofangkun**

### CODE REVIEW

File: angular/pom.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
     <artifactId>zeppelin</artifactId>
     <groupId>org.apache.zeppelin</groupId>
     <version>0.6.0-incubating-SNAPSHOT</version>
+    <relativePath>..</relativePath>
```
Comment (Leemoonsoo) [inline angular/pom.xml]: While parent's pom.xml is in parent directory, i think it's safe to omit `<relativePath>..</relativePath>`. Is there any special reason explicitly defining it?

File: pom.xml [hunk 2]
```diff
@@ -104,7 +104,7 @@
   <properties>
     <slf4j.version>1.7.10</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
-    <libthrift.version>0.9.0</libthrift.version>
+    <libthrift.version>0.9.2</libthrift.version>
```
Comment (Leemoonsoo) [inline pom.xml]: Could you share a reason why you decided to change libthrift version?

Comment (caofangkun) [inline pom.xml]: @Leemoonsoo 
 trivial hash function(always returning 0 eg: [RemoteInterpreterContext.java#L582](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift/RemoteInterpreterContext.java#L582)) could be used if that's generic implementation thrift should provide.

File: angular/pom.xml (hunk 1 above)
Comment (caofangkun) [inline angular/pom.xml]: @Leemoonsoo Yes ,you are right, it's safe to omit , explicitly defining is much clearer for reading and no more special reason,

**Comment by Leemoonsoo** [comment]
Thanks for the explanation. LGTM!

**Pull request closed without merging**